docs/dev/table/sql.md

```diff
@@ -1434,6 +1434,79 @@ If the table does not exist, nothing happens.
 
 {% top %}
 
+### Alter Table
+
+* Rename Table
+
+{% highlight sql %}
+ALTER TABLE [catalog_name.][db_name.]table_name RENAME TO new_table_name
```

Comment:
do we support [IGNORE IF NOT EXISTS] here?

Reply:
We don't support  [IGNORE IF NOT EXISTS] here